SWLOR.Game.Server/Service/AbilityService/RecastGroup.cs

```diff
@@ -25,14 +25,14 @@ public enum RecastGroup
         MindTrick = 8,
         [RecastGroup("Force Burst", "Force Burst", true)]
         ForceBurst = 9,
-        [RecastGroup("Force Body", "Force Body", true)]
-        ForceBody = 10,
+        [RecastGroup("Force Restore", "Force Restore", true)]
+        ForceRestore = 10,
         [RecastGroup("Force Drain", "Force Drain", true)]
         ForceDrain = 11,
         [RecastGroup("Force Lightning", "F. Lightning", true)]
         ForceLightning = 12,
-        [RecastGroup("Force Mind", "Force Mind", true)]
-        ForceMind = 13,
+        [RecastGroup("Grenades", "Grenades", true)]
+        Grenades = 13,
```

Comment:
I don't understand this change. If Force Mind was removed, fine, but all of the grenades have their own recast timers now. Please check and verify, please explain to me if I'm off base.

Reply:
Grenades have a short global cooldown and longer indivodual cooldowns.